### PR TITLE
Remap TextStyle to its own type

### PR DESCRIPTION
**Pull request by eimantas**

### CODE REVIEW

File: ios/Sources/MaaS/UI/Components/Button.swift [hunk 1]
```diff
@@ -79,7 +79,7 @@ struct Button_Previews: PreviewProvider, Snapped {
                     .environment(\.uiColorPrimary, .systemBlue)
                     .environment(\.uiColorOnPrimary, .systemYellow)
                     .environment(\.cornerRadiusButton, 20)
-                    .environment(\.uiFontTextL, UIFont(name: "Papyrus", size: 25)!)
+                    .environment(\.textStyleTextL, TextStyle(fontWeight: .normal, fontStyle: .normal, fontSize: 25, lineSpacing: 0, color: nil))
```
Comment (justasm) [inline ios/Sources/MaaS/UI/Components/Button.swift:82]: Seems like we'll want font in `TextStyle`, too 🙈

File: ios/Sources/MaaSTheme/Kotlin.swift [hunk 2]
```diff
@@ -1,14 +1,19 @@
 import UIKit
 import SwiftUI
+import MaasCore
 
 @dynamicMemberLookup
 public struct Kotlin<T> {
 
-    let value: T
+    public let value: T
```
[the remark this answers is not in the document]
Comment (mikasdo) [reply ios/Sources/MaaSTheme/Kotlin.swift:8]: nope :)

File: ios/Sources/MaaSTheme/Kotlin.swift [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
     public init(_ value: T) { self.value = value }
 
-    public subscript(dynamicMember keyPath: KeyPath<T, UIFont>) -> Font {
-        Font(value[keyPath: keyPath])
+    public subscript<V>(dynamicMember keyPath: KeyPath<T, V>) -> V {
+        value[keyPath: keyPath]
```
Comment (justasm) [inline ios/Sources/MaaSTheme/Kotlin.swift:12]: This is to allow pass-through of values that don't need mapping?

**Review by justasm — APPROVED** [review]

**Review by mikasdo — APPROVED** [review]

File: ios/Sources/MaaSTheme/Kotlin.swift (hunk 3 above)
Comment (eimantas) [reply ios/Sources/MaaSTheme/Kotlin.swift:12]: Pretty much. Also, AFAIK, to allow optionality.

Comment (eimantas) [reply ios/Sources/MaaSTheme/Kotlin.swift:12]: Nope. there's one more subscript below 🙈

**Pull request merged**